networkParams/src/main/AndroidManifest.xml

```diff
@@ -3,5 +3,6 @@
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
```

Comment:
remove this permission, we didn't need it.

Reply:
Done